src/main/java/frc/robot/OperatorBoard.java

```diff
@@ -0,0 +1,119 @@
+package frc.robot;
+
+import edu.wpi.first.wpilibj.Joystick;
+import edu.wpi.first.wpilibj2.command.button.Button;
+
+/**
+ * A wrapper class for our custom-made operator board. Does not have a
+ * constructor. Access statically. Access Buttons via public static state.
+ * Access axis values via methods. Use methods to make buttons light up.
+ */
+public class OperatorBoard {
+
+    private static Joystick HARDWARE;
+    public OperatorBoardButton BUTTON_ONE;
+    public OperatorBoardButton BUTTON_TWO;
+    public OperatorBoardButton BUTTON_THREE;
+    public OperatorBoardButton BUTTON_SIX;
+    public OperatorBoardButton BUTTON_ELEVEN;
+    public OperatorBoardButton BUTTON_TWELVE;
+    public OperatorBoardButton BUTTON_THIRTEEN;
+    public OperatorBoardButton BUTTON_FOURTEEN;
+
+    private static OperatorBoard instance;
```

Comment:
```suggestion
    private static final OperatorBoard instance = new OperatorBoard();
```
why not do this instead of an if null check?

Reply:
Well the null check is how the CommandScheduler in wpilib does it.

Reply:
i don't get that. and why do you only want to construct the operator board once